### PR TITLE
Fix grafana func serializers

### PR DESCRIPTION
**Pull request by jasminetealover**
In grafana there can be collapsable group of panels. Let's add it in here by extending `Rowset` class. Also fixed some typos for grafana. And added ability to have `sum` without label (which can be handy when we want to sum all the series)

### CODE REVIEW

File: yt/admin/dashboards/yt_dashboards/common/postprocessors.py [hunk 1]
```diff
@@ -94,6 +94,12 @@ def postprocess(self, tags, sensor_name=None):
             tags[GrafanaSystemTags.Rate] = True
             sensor_name = sensor_name[:-5]
         sensor_name = sensor_name.replace(".", "_")
+
+        if SystemFields.LegendFormat in tags:
+            value = tags[SystemFields.LegendFormat]
+            value = value.replace(ContainerTemplate, "{{pod}}")
```
Comment (jasminetealover) [inline yt/admin/dashboards/yt_dashboards/common/postprocessors.py:100]: `ContainerTemplate -> {{pod}}` is very YT specific replacement, makes sense to have it in yt postprocessor

**Comment by Kontakter** [comment]
This comment is not for this specific PR, but it seems that we need to add unit tests for the grafana and monitoring serializers

**Review by Kontakter — CHANGES_REQUESTED** [review]

File: yt/admin/dashboard_generator/yt_dashboard_generator/specific_sensors/monitoring.py [hunk 2]
```diff
@@ -136,8 +139,14 @@ def value(self, key, value, overwrite=True):
...
         if hasattr(arg, "serialize"):
             return arg.serialize(default_serializer)
-        else:
-            return default_serializer(arg)
+
+        if isinstance(arg, list):
+            serialized_elements = [
+                default_serializer(item) for item in arg
+            ]
+            return ", ".join(serialized_elements)
```
[the remark this answers is not in the document]
Comment (jasminetealover) [reply yt/admin/dashboard_generator/yt_dashboard_generator/specific_sensors/monitoring.py]: good catch!

**Comment by jasminetealover** [comment]
> This comment is not for this specific PR, but it seems that we need to add unit tests for the grafana and monitoring serializers

Yes. This would be very convenient. It's currently almost impossible to test monitoring generators.

**Review by Kontakter — APPROVED** [review]

**Pull request closed without merging**